merlin/systems/dag/ops/feast.py

```diff
@@ -176,6 +247,19 @@ def export(self, path, input_schema, output_schema, params=None, node_id=None, v
         return super().export(path, input_schema, output_schema, self_params, node_id, version)
 
     def transform(self, df: InferenceDataFrame) -> InferenceDataFrame:
+        """
+        Transform input dataframe to output dataframe using function logic.
```

Comment:
s/dataframe/DataFrame/, I think.

In the foll desc, is "...dictionary, the data to manipulate." true? (Avoid future tense, avoid conjugations of to be.)